plugin/keyring_vault/vault_parser_composer.cc

```diff
@@ -314,6 +316,20 @@ bool Vault_parser_composer::parse_key_data(const Secure_string &payload,
   }
 
   if (vault_version == Vault_version_v2) {
+    it = data_node->FindMember(metadata_key);
+
+    if (it != data_node->MemberEnd() && it->value.IsObject()) {
+      const auto *metadata_node = &it->value;
+      it = metadata_node->FindMember(deletion_time_key);
+
+      if (it != metadata_node->MemberEnd() && it->value.IsString() &&
+          strlen(it->value.GetString()) != 0) {
+        logger->log(MY_ERROR_LEVEL,
```

Comment:
This could be a pretty common case. May be `MY_ERROR_LEVEL` is too much. Consider something like WARNING or INFO.

Reply:
Done